content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,87 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. You no longer need to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Envoy proxy readiness checks. They now check Envoy's readiness status.
```

Comment:
very nit - we always had readiness checks, we just added an additional one. maybe change to

> They now also check

Reply:
accepted.